js/main.js

```diff
@@ -9,6 +9,74 @@ var overAllScoreElement = document.querySelector('.overall-score');
 var xIcon = document.querySelector('.fa-xmark');
 var summaryButton = document.querySelector('#summary-button');
 var modalContainer = document.querySelector('.modal-container');
+var backToSearchButton = document.querySelector('.back-to-search-container').querySelector('button');
+var favSkipContainer = document.querySelector('.fav-skip-container');
+
+function removeStatsList() {
+  cityStatsCityStatsList.innerHTML = '';
+}
+
+function notificationPopUp() {
+  var notificationContainer = document.querySelector('.notification-container');
```

Comment:
You seem to be querying for this element a lot of times, I would make a global variable for it so you can just reuse that variable.